tests/integration/test_vulnerability_detector/test_scan_results/test_scan_vulnerability_removal.py

```diff
@@ -107,7 +106,7 @@ def test_vulnerability_removal_update_package(configuration, metadata, set_wazuh
         - Force again a full scan and wait for the full scan event log.
         - Check that vulnerability removal has been detected (in log) and check for the removal alert.
 
-    wazuh_min_version: 4.2.0
+    wazuh_min_version: 4.4.0
```

Comment:
Update `test_phases` documentation too, due to now the system is mocked too

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/2847/commits/4e24082226514562f4172e7d4ea75e848e6b4b56